src/plugins/dashboard/public/application/embeddable/services/direct_query_sync_services.ts

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
should this be under `direct_query_sync/` you added?

Reply:
makes sense. moved